portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,23 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+async function showServerTime() {
+    const responseFromServer = await fetch('/date');
+    const textFromResponse = await responseFromServer.text();
+  
+    const dateContainer = document.getElementById('date-container');
+    dateContainer.innerText = textFromResponse;
+  }
+
+  async function test() {
```

Comment:
Rename function.

Always name your functions and variables with intention.

also, remove dead code

Reply:
And rename local vars, e.g. "hi"

Reply:
Fixed!